cob_obstacle_distance/launch/example_arm_right.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
+    <node name="cob_obstacle_distance" pkg="cob_obstacle_distance" type="cob_obstacle_distance" output="screen">
+        <rosparam param="joint_names">[arm_right_1_joint, arm_right_2_joint, arm_right_3_joint, arm_right_4_joint, arm_right_5_joint, arm_right_6_joint, arm_right_7_joint]</rosparam>
+        <rosparam param="robo_namespace">arm_right</rosparam>
+        <rosparam param="chain_base_link">arm_right_base_link</rosparam>
+        <rosparam param="chain_tip_link">arm_right_7_link</rosparam>
```

Comment:
That's fine. robo_namespace will be deleted and ns added. 

But the intention here is to have a example launch file (as the name says) to be started without any dependencies on other launch-files which are hard to find. 

With the same names cob_obstacle_distance is possible to be started with the global launch-files as well as the example file.